### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by raddevon**
Add CLI version field and instructions for how to get the version numbers

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -3,7 +3,13 @@ name: Bug report
 about: Create a report to help us improve
 ---
 <!-- Please search existing issues to avoid creating duplicates. -->
+
+<!--
+For the EdgeDB Version: run `edgedb` from your project directory copying the first line of output or query `select sys::get_version_as_str();`
```
Comment (elprans) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: You can get the version as a oneliner: `edgedb query 'select sys::get_version_as_str()'`

Comment (raddevon) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks, @elprans. I circled back and updated this across the new and updated issue templates.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -2,8 +2,16 @@
 name: Bug report
 about: Create a report to help us improve
 ---
+
 <!-- Please search existing issues to avoid creating duplicates. -->
+
+<!--
+For the EdgeDB Version: run `edgedb query 'select sys::get_version_as_str()'` from your project directory or query `select sys::get_version_as_str();`
+For the EdgeDB Client Version: Run `edgedb --version` from anywhere
```
Comment (elprans) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
For the EdgeDB CLI Version: Run `edgedb --version` from anywhere
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -2,8 +2,16 @@
 name: Bug report
 about: Create a report to help us improve
 ---
+
 <!-- Please search existing issues to avoid creating duplicates. -->
+
+<!--
+For the EdgeDB Version: run `edgedb query 'select sys::get_version_as_str()'` from your project directory or query `select sys::get_version_as_str();`
```
Comment (elprans) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```suggestion
For the EdgeDB Version: run `edgedb query 'select sys::get_version_as_str()'` from your project directory (or run `select sys::get_version_as_str();` in the EdgeDB interactive shell).
```

**Review by elprans — APPROVED** [review]

**Pull request merged**